extensions/grpc/protoc/src/main/resources/MutinyInterface.mustache

```diff
@@ -2,10 +2,12 @@
 package {{packageName}};
 {{/packageName}}
 
+import io.quarkus.grpc.runtime.MutinyService;
+
 @javax.annotation.Generated(
 value = "by {{classPrefix}} Grpc generator",
 comments = "Source: {{protoName}}")
-public interface {{serviceName}} {
+public interface {{serviceName}} extends MutinyService {
```

Comment:
I know it's an empty interface, but ... why? Do we need it for internal reasons, or does it help users in some way?

Reply:
We needs this internally, to be able to collect all the generated service interfaces. We could add an annotation but a marker interface is IMO better. We could add some functionality later.

Reply:
I don't think a user would ever notice the interface is there...

Reply:
I totally understand your crusade for clear API but we must be practical too :-).

Reply:
I just don't like it :-) And I'm pretty sure we can communicate the full set of generated interfaces through other means, because it's us who generate them, no? Like, I mean, a `BuildItem`? :-)

Reply:
In other words -- if this is the only thing I can complain about, you did a pretty great job :-)

Reply:
>  Like, I mean, a `BuildItem`? :-)

The problem is that these bits are not generated by a build step (gizmo, etc.) but a `io.quarkus.deployment.CodeGenProvider` that is executed during the `generate-sources` phase of the build.

Reply:
Ah ah, very good point. I guess you convinced me there's no other way (but I still don't like it :-) ).